spec/requests/api/v1/topics/index_spec.rb

```diff
@@ -18,9 +18,12 @@
     end
 
     it 'returns all the topics' do
-      subject
       expect(json[:topics].pluck(:id, :name)).to match_array(topics.pluck(:id, :name))
     end
+
+    it 'returns the image_url in the body response' do
+      json[:topics].each { |topic| expect(topic).to have_key('image_url') }
```

Comment:
Maybe you can test for the presence of the right image_url `value` instead of just the presence of the key. 

This test won't fail if the image_url value that is returned is wrong (as long as any image_url value is returned in the response, it'll pass)